src/index.html

```diff
@@ -13,6 +13,43 @@
     />
```

Comment:
The `href` attribute in the `<link>` tag should reference a CSS file, not an SCSS file. Change `href="./styles/index.scss"` to `href="./styles/index.css"` to ensure the styles are applied correctly.